### ISSUE TITLE
what's the source of datasets? Please specify it

### ISSUE THREAD

**Issue by rahuls321** [lakshmanmallidi/SleepApneaDetection#1]

**Comment by nitinsaikumar** [comment]
the link to the datasets was provided int the readme.txt

**Comment by Surenborah234** [comment]
How to run the program?
means should i use matlab or jupyter

**Comment by eladwass** [comment]
How did you collect the dataset? How did you labeled the dataset? Did you reproduce an article or something?

**Comment by fulincai** [comment]
Yeah, I got the same question on this. If you can kindly provide the source of this dataset, we can correctly cite this work. Thanks!

**Comment by MuhammadTalmeez337** [comment]
Can you provide the source of data? From where do you get this data? Is this a reliable source?
